### DISCUSSION TITLE
Add an extra level to resolvers

### DISCUSSION THREAD

**Discussion by janwillemvj** [MichalLytek/type-graphql#911] [Q&A]
I would like to know if it is possible to add a level to my resolvers where I specify the Entity?

Currently I have a mutation to create a new company as follow:

```graphQL
mutation {
    createCompany(arguments) {
        uuid
        title
        status
        startDate
        endDate
        legislation
   }
}
```

But I am wondering if its possible to have a structure as follow:

```graphQL
mutation {
    company{
        create(arguments) {
            uuid
            title
            status
            startDate
            endDate
            legislation
        }
    }
}
```

This way I can have generic resolvers for my CRUD operations and use them for all Entities with similar implementations

**Comment by MichalLytek** [comment]
#64

**Reply by janwillemvj** [reply]
Thank you for the quick response, and the detailed explanation in the attached thread. 

Keep doing what you are doing! Awesome framework!
